### PR TITLE
Add ESLint

### PR DESCRIPTION
**Pull request by syossan27**
It is a measure to prevent recurrence of problems like #35 .

### CODE REVIEW

**Comment by geowarin** [comment]
Thanks!
Could you also add this to the test script in package.json so that it could be triggered by CI ?

**Comment by syossan27** [comment]
Oh! It is also necessary.
I will do it later.

**Comment by syossan27** [comment]
@geowarin Done:-)

File: package.json [hunk 1]
```diff
@@ -4,7 +4,7 @@
   "description": "Recognizes certain classes of webpack errors and cleans, aggregates and prioritizes them to provide a better Developer Experience",
   "main": "index.js",
   "scripts": {
-    "test": "jest"
+    "test": "eslint --ignore-pattern test/* & jest"
```
Comment (christophehurpeau) [inline package.json]: `&&` might be best ? else if eslint fails, the test won't fail because of return status ?

Comment (syossan27) [inline package.json]: @christophehurpeau
Sorry...Thanks for the advice!
I will fix it.

**Pull request merged**